tests/nested/manual/core20-da-lockout/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: Check that an unclean shutdown on UC20 devices works
+
+details: |
+    This test checks that an unclean shutdown of a UC20 device does
+    not cause an eventual DA lockout (LP: 1979185)
+
+systems: [ubuntu-20.04-64, ubuntu-22.04-64]
+
+# test takes a long time
+kill-timeout: 2h
+
+environment:
+  NESTED_ENABLE_TPM/encrypted: "true"
+  NESTED_ENABLE_SECURE_BOOT/encrypted: "true"
+
+execute: |
+    tests.nested build-image core
+    tests.nested create-vm core
+
+    tests.nested exec "sudo snap wait system seed.loaded"
+    tests.nested wait-for device-initialized
+
+    # XXX: this test takes forever
```

Comment:
This test takes more than 1h, it's not finished yet so it's a bit unclear if we want to mere this but it is a nice validation to double check that the problem really went away.